deel-2-flexbox-layout/opdracht-5/styles/styles.css

```diff
@@ -1 +1,42 @@
+/*Schrijf hier jouw CSS*/
+
+body {
+    display: flex;
+    flex-direction: column;
+    height: 100vh;
+}
+
+main {
+    display: flex;
+    flex-basis: 900px;
```

Comment:
toevoegen: flex-direction: row (main valt binnend de body en die staat op column, dus moet de direction hier opnieuw gedefineerd worden) door hem een row mee te geven zorg je ervoor dat de tekst in het article op de juiste manier weergegeven wordt